modules/rostests/apitests/ntdll/LdrFindResource_U.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Test for LdrFindResource_U
+ * COPYRIGHT:   Copyright 2025 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#include "precomp.h"
+
+static void Test_CORE_20401(void)
+{
+    HMODULE hmod = GetModuleHandleW(NULL);
+    LDR_RESOURCE_INFO info;
+    IMAGE_RESOURCE_DATA_ENTRY *entry = NULL;
+    NTSTATUS Status;
+
+    // Use LdrFindResource_U to find a bitmap resource called "NORMAL_FRAMECATION_BMP"
```

Comment:
Suggestion:

Change "NORMAL_FRAMECATION_BMP" to "NORMAL_FRAMECAPTION_BMP".

Reply:
Yeah it's just typo in the comment, the variable itself is correct.

Reply:
Fixed